README.md

```diff
@@ -1 +1,52 @@
-# homebrew-versions
\ No newline at end of file
+# Helm and Kubernetes-cli legacy versions
+
+Attempt to rebuild previous versions of clients needed for communication with existing older production infrastructure
+
+
+## Update with new formula
+
+When new version of Helm or Kubernetes-cli is released, brew will update its packages. To view all the previous versions of the tool, we can visit the brew histories of [Helm](https://github.com/Homebrew/homebrew-core/commits/7877b4bbcfa48028349a5cbed0d2f6ef5dbcc84b/Formula/helm.rb) and [Kubernetes-cli](https://github.com/Homebrew/homebrew-core/commits/674e098c3efce9196d91397d325c66d488aacc6b/Formula/kubernetes-cli.rb) to view list of all the previous versions.
+
+
+### Updating tap
+
+1. Create a file `helm@<x.x.x>.rb` or `Kubernetes-cli@<x.x.x>`
```

Comment:
Why is 1 an `.rb` file and the other have no extension?
Is this significant? Is it a typo?

Reply:
it's typo, will fix it

Reply:
Fixed